db/seeds.d/111-upgrade_tasks.rb

```diff
@@ -2,6 +2,7 @@
   [
     {:name => 'katello:correct_repositories', :long_running => true, :skip_failure => true, :always_run => true},
     {:name => 'katello:correct_puppet_environments', :long_running => true, :skip_failure => true, :always_run => true},
-    {:name => 'katello:clean_backend_objects', :long_running => true, :skip_failure => true, :always_run => true}
+    {:name => 'katello:clean_backend_objects', :long_running => true, :skip_failure => true, :always_run => true},
+    {:name => 'katello:clear_checksum_type'}
```

Comment:
that should be  katello:upgrades:3.8:clear_checksum_type since thats the task name.